validator/client/runner.go

```diff
@@ -64,9 +64,9 @@ func run(ctx context.Context, v Validator) {
 			span.AddAttributes(trace.Int64Attribute("slot", int64(slot)))
 			slotCtx, cancel := context.WithDeadline(ctx, v.SlotDeadline(slot))
 			// Report this validator client's rewards and penalties throughout its lifecycle.
+			log := log.WithField("slot", slot)
 			if err := v.LogValidatorGainsAndLosses(slotCtx, slot); err != nil {
-				log.Errorf("Could not report validator's rewards/penalties for slot %d: %v",
-					slot, err)
+				log.WithError(err).Error("Could not report validator's rewards/penalties")
```

Comment:
we should still state the slot of the error

Reply:
Slot is set earlier in the function, to reduce duplication of code.